d2/src/main/java/com/linkedin/d2/balancer/util/hashing/MPConsistentHashRing.java

```diff
@@ -53,6 +53,16 @@ public class MPConsistentHashRing<T> implements Ring<T>
   public static final int DEFAULT_POINTS_PER_HOST = 1;
 
   private static final Logger LOG = LoggerFactory.getLogger(ConsistentHashRing.class);
+  static {
+    try {
+      LongHashFunction.class.getMethod("xx_r39", long.class);
+    } catch (NoSuchMethodException ex) {
+      // SI-36497
+      LOG.error("Required method xx_39 not found, this means an supported version of the "
+          + "zero-allocation-hashing library is being used. Do not use later than 0.7 if you want to use pegasus", ex);
```

Comment:
"xx_39" should be "xx_r39"? "supported" should be "unsupported", right?

Reply:
Thanks for catching that, fixed.